bigtable/admin_test.go

```diff
@@ -56,18 +65,54 @@ func setupTableClient(t *testing.T, ac btapb.BigtableTableAdminClient) *AdminCli
 	return c
 }
 
-func TestTableAdmin_UpdateTable(t *testing.T) {
+func TestTableAdmin_CreateTableFromConf_DeletionProtection_Protected(t *testing.T) {
+	mock := &mockTableAdminClock{}
+	c := setupTableClient(t, mock)
+
+	deletionProtection := Protected
+	err := c.CreateTableFromConf(context.Background(), &TableConf{TableID: "My-table", DeletionProtection: deletionProtection})
+	if err != nil {
+		t.Errorf("CreateTableFromConf failed: %v", err)
+	}
+	createTableReq := mock.createTableReq
+	if !cmp.Equal(createTableReq.TableId, "My-table") {
+		t.Errorf("CreateTableRequest does not match, TableID: %v", createTableReq.Table.Name)
```

Comment:
do you mean: createTableReq.TableId?

Maybe mention what is not matching, in this case the table ID.

Maybe something like this: "t.Errorf("Unexpected table ID in the request: %v, expected %v", createTableReq.TableId, "My-table").